server/views/formPages/supervisor/recatReview.html

```diff
@@ -79,15 +79,18 @@ <h1 class="govuk-heading-l openConditionsHeader">Open Conditions</h1>
       {
         value: "R",
         id: "overriddenCategoryR",
-        checked: true,
+        checked: data.supervisor.review.supervisorOverriddenCategory === 'R',
         text: "Consider for closed",
         R: true
       },
       {
         value: "T",
```

Comment:
Please consider switching order of female open.closed with YOI open/closed to match the ticket.

Reply:
Don't understand there is no screenshot and there is nothing on the ticket stating this for the supervisor page.